Library/Homebrew/compat/macos.rb

```diff
@@ -80,7 +80,6 @@ def gcc_42_build_version
     alias_method :gcc_build_version, :gcc_42_build_version
 
     def llvm_build_version
-      DevelopmentTools.llvm_build_version
```

Comment:
It's a compatibility method hence: to avoid breaking the DSL. The previous invocation can return `nil` so this is an acceptable value.